docs/sitelist.md

```diff
@@ -271,6 +271,7 @@ The above is a reference list. Please see [the manualsearch doc](./manualsearch.
 + #### First Anal Quest | Matching type: *[Limited](./manualsearch.md#limited-search)*
 + #### Fitting-Room | Matching type: *[Exact](./manualsearch.md#exact-match)* - **SceneID (as listed in URL), Date Add**
 + #### FuckedHard18 | Matching type: *[Enhanced](./manualsearch.md#enhanced-search)*
++ #### FuckingAwesome | Matching type: *[Enhanced](./manualsearch.md#enhanced-search)* - **Title, Actor and Date**
```

Comment:
You can remove "Title, Actor, and Date" as Title/Actor Search with Date Match is automatically assumed when the site is listed as Enhanced.

Reply:
ok